far/common/preprocessor.hpp

```diff
@@ -124,13 +124,13 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 	NOT_COPY_ASSIGNABLE(Type)
 
 #define MOVE_CONSTRUCTIBLE(Type) \
-	Type(Type&&) = default
+	Type(Type&&) noexcept = default
```

Comment:
[Please don't](https://stackoverflow.com/questions/18653726/is-the-default-move-constructor-defined-as-noexcept).